packages/teams-js/src/private/videoEx.ts

```diff
@@ -84,12 +88,30 @@ export namespace videoEx {
    * @internal
    * Limited to Microsoft-internal use
    */
-  export type VideoFrameCallback = (
+  export type SharedFrameCallback = (
     frame: VideoFrame,
     notifyVideoFrameProcessed: () => void,
     notifyError: (errorMessage: string) => void,
   ) => void;
 
+  /**
+   * @hidden
+   * @beta
+   * Callbacks and configuration supplied to the host to process the video frames.
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export type RegisterForVideoFrameParameters = {
+    /**
+     * Callback function to process the video frames shared by the host.
+     */
+    sharedFrameCallback: SharedFrameCallback;
```

Comment:
@TrevorJoelHarris We'll add mediaStream support in videoEx in another change, may need another design review because videoEx not only have video frame but also audioInferenceResult. We're testing the performance of the design, will start a design review with you when it's ready.